wagtail/admin/views/reports/scheduled_pages.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+import django_filters
+from django.core.exceptions import PermissionDenied
+from django.utils.translation import gettext_lazy as _
+
+from wagtail.admin.filters import DateRangePickerWidget, WagtailFilterSet
+from wagtail.admin.views.reports.base import PageReportView
+from wagtail.admin.views.scheduled_pages import get_scheduled_pages_for_user
+from wagtail.core.models import Page, UserPagePermissionsProxy
+
+
+class ScheduledPagesReportFilterSet(WagtailFilterSet):
+    go_live_at = django_filters.DateFromToRangeFilter(widget=DateRangePickerWidget)
```

Comment:
perhaps `DateTimeFromToRangeFilter` is a more appropriate filter given that go live happens at a specific date and time?

Reply:
I know this is expanding the scope, but we do want at least another filter (by page type). ie. I want to publish the matching press releases that are scheduled for publishing